app/lib/actions/katello/content_view/incremental_update/sync_and_install_content.rb

```diff
@@ -0,0 +1,32 @@
+module Actions
+  module Katello
+    module ContentView
+      module IncrementalUpdate
+        class SyncAndInstallContent < ::Actions::EntryAction
+          def plan(cv_env_map: {}, host_ids: [], errata_ids: [])
+            sequence do
+              concurrence do
+                cv_env_map.each_pair do |content_view_id, environment_ids|
+                  environment_ids.each do |env_id|
+                    content_view = ::Katello::ContentView.find(content_view_id)
+                    environment = ::Katello::KTEnvironment.find(env_id)
+
+                    plan_action(::Actions::Katello::ContentView::CapsuleSync, content_view, environment)
+                  end
+                end
+              end
```

Comment:
we may want to test what happens if a capsule can't be synced due to either:
a) it being down at the start of the sync
b) the sync failing for some reason after its started

I think for 
a) the whole task may abort which we probably don't want, we likely want best effort across all capsules
b) I *think* that the sync would be skipped and everything else would occur (Because CapsuleContent::Sync has a rescue_strategy of Skip), but we may want to double check that in reality.

Reply:
The incremental update completes with a warning but other syncs complete :+1: